web/src/information_density.ts

```diff
@@ -0,0 +1,21 @@
+import $ from "jquery";
+
+import {page_params} from "./base_page_params";
+import {user_settings} from "./user_settings";
+
+export function set_base_typography_css_variables(): void {
+    if (!page_params.development_environment) {
+        return;
+    }
```

Comment:
Why do we need the development check? The default values in the database should match the legacy default values, right?

Reply:
Oh, yup. Good thinking--I was trying to avoid disturbing anything at all outside of dev, but this is unnecessary.